weave/bin/modules/cli_migrate.py

```
@@ -0,0 +1,411 @@
+    """
+    verbose = ctx.obj.get('VERBOSE', False)
+    
+    if verbose:
+        console.print(f"[blue]Rolling back migrations for: {database}[/blue]")
+    
+    try:
+        action = 'downgrade'
+        if revision:
+            action = f'downgrade {revision}'
+        else:
+            action = 'downgrade -1'  # Rollback one migration
```

Ruff (0.11.9)

Use ternary operator `action = f'downgrade {revision}' if revision else 'downgrade -1'` instead of `if`-`else`-block

(SIM108)